docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -223,6 +266,30 @@ tasks
 </tab>
 </tabs>
 
+#### Example of setting of JVM target validation mode
```

Comment:
I think this should be located in this documentation section: https://kotlinlang.org/docs/gradle-configure-project.html#check-for-jvm-target-compatibility-of-related-compile-tasks